adafruit_hue.py

```diff
@@ -115,11 +115,11 @@ def discover_bridge(self):
             json_data = resp.json()
             bridge_ip = json_data[0]["internalipaddress"]
             resp.close()
-        except:
+        except Exception as e:
```

Comment:
Should this be `except TypeError as e:`? I feel like that's what I've seen previous, but I could be mistaken.

Reply:
@kattni I'm not really sure

Reply:
I'm good with it as it is then, as long as we can get it passing.